src/App.tsx

```diff
@@ -26,70 +27,132 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
+  let resultList;
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case SortType.ALPABET:
+      resultList = visibleGoods.sort((goodA, goodB) => (
+        goodA.localeCompare(goodB)
+      ));
+      break;
 
-  return visibleGoods;
+    case SortType.LENGTH:
+      resultList = visibleGoods.sort((goodA, goodB) => (
+        goodA.length - goodB.length
+      ));
+      break;
+    default:
+      resultList = visibleGoods;
+      break;
+  }
```

Comment:
You can put switch case into sort method, so you don't need to copy `sort` all of the time

Reply:
agree with @grafit-off